src/Appwrite/Detector/Detector.php

```diff
@@ -47,7 +47,7 @@ public function getOS(): array
      */
     public function getClient(): array
     {
-        if (strpos($this->userAgent, 'Appwrite CLI') !== false) {
+        if (strpos($this->userAgent, 'AppwriteCLI') !== false) {
```

Comment:
Just want to confirm whether this change is intentional or not since only a whitespace has been removed here?

Reply:
Yes this is intentional :)